class AuthenticationError(BaseException):
    pass


class NotFoundError(BaseException):
    pass
